charts/evm-rollup/values.yaml

```diff
@@ -172,6 +172,8 @@ config:
     # The expected fastest block time possible from sequencer, determines polling
     # rate.
     sequencerBlockTimeMs: 2000
+    # The expected fastest block time possible from DA, determines polling rate.
+    celestiaBlockTimeMs: 12000
```

Comment:
Probably update this default.

Reply:
Should update `celestia-local` blocktime as well then